src/uct/ib/base/ib_md.c

```diff
@@ -120,7 +120,7 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "For example a filter for the default subnet prefix can be specified as: fe80:0:0:0",
      ucs_offsetof(uct_ib_md_config_t, subnet_prefix), UCS_CONFIG_TYPE_STRING},
 
-    {"GPU_DIRECT_RDMA", "try",
+    {"GPU_DIRECT", "try",
```

Comment:
@yosefe I think we should have RDMA in the name. otherwise term "gpu direct" reflects broader capabilities including peer-to-peer transfers between the GPUs. ( https://developer.nvidia.com/gpudirect).  This option is just specific to RDMA capability between GPUs and other PCIe device(HCA).